extensions/dacpac/src/wizard/api/dacFxConfigPage.ts

```diff
@@ -192,8 +193,9 @@ export abstract class DacFxConfigPage extends BasePage {
 		// use previous file location if there was one
 		if (this.fileTextBox.value && path.dirname(this.fileTextBox.value)) {
 			return path.dirname(this.fileTextBox.value);
-		} else { // otherwise use the folder open in the Explorer or the home directory
-			return vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0].uri.fsPath : os.homedir();
+		} else { // otherwise use the default save location setting or the home directory
+			const defaultLocation: vscode.Uri = defaultSaveLocation();
+			return defaultLocation ? defaultLocation.fsPath : os.homedir();
```

Comment:
I think it makes sense for this logic to be down in `defaultSaveLocation()` to consolidate what the logic is.